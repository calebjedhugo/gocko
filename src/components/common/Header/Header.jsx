import React from "react";
import Typography from "@material-ui/core/Typography";

const Header = ({ children }) => (
    <Typography variant="h2">{children}</Typography>
);

export default Header;
