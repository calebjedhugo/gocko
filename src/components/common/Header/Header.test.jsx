import { render } from "@testing-library/react";
import Header from "./Header";

describe("<Header>", () => {
    it("Matches previous snapshot", () =>
        expect(
            render(<Header>Awesome Header</Header>).baseElement
        ).toMatchSnapshot());
});
