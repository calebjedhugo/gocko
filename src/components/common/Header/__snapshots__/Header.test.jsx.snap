// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> Matches previous snapshot 1`] = `
<body>
  <div>
    <h2
      class="MuiTypography-root MuiTypography-h2"
    >
      Awesome Header
    </h2>
  </div>
</body>
`;
